### DISCUSSION TITLE
Capture RAISE LOG/WARNING/etc from plpgsql functions using Prisma.TransactionClient

### DISCUSSION THREAD

**Discussion by renatocron** [prisma/prisma#15943] [Q&A] [answered]
Hello,
I'm running some functions called using Prisma.TransactionClient.$queryRaw

Is there any way to capture the notices the database emits? This should be easy to get when using the TransactionClient as there's already a reserved connection to this

**Comment by austincrim** [comment, marked as answer]
Hey @renatocron 👋,

This is not yet possible with Prisma, though it sounds like a good candidate for a [feature request](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=). 

Please open one with the relevant details, and our engineers will take a look!

**Reply by anitacung** [reply]
👋 Hi @renatocron,

Have you had the chance to create a new feature request yet? 

We appreciate that you already created a feature request for us 
here: https://github.com/prisma/prisma/issues/15886 and suggested a solution/alternative. Thank you so much for your feedback! 

If there is anything else we can help you with, I’d love to get you in touch with the product team.

**Reply by renatocron** [reply]
Hi @anitacung,

I did not created yet this feature request, I will open late today, but this one is usually just useful when using queryRaw/executeRaw

I keep following the releases, but I did not had time yet to read other discussions to see what's going on and what's the vibe for new features that is less 'orm-like': eg, raw values on update/create

**Reply by nurul3101** [reply]
Please feel free to let us know in case you have any feedback for us 🙂

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
